src/Symfony/Bridge/Twig/NodeVisitor/TranslationDefaultDomainNodeVisitor.php

```diff
@@ -114,6 +114,6 @@ private function isNamedArguments(Node $arguments): bool
 
     private function getVarName(): string
     {
-        return sprintf('__internal_%s', hash('sha256', uniqid(mt_rand(), true), false));
+        return sprintf('__internal_%s', hash('xxh128', uniqid(mt_rand(), true)));
```

Comment:
I don't think we need a random var name here. A global sequence would be enough to ensure there is no conflict.
Similar to what I did in https://github.com/twigphp/Twig/pull/3601/files#diff-47bc8653b21716576958e25b6d7356ecb0f0070f17554d12d2bee985ac211b26R59